cluster/gce/gci/configure-helper.sh

```diff
@@ -409,6 +409,23 @@ function assemble-docker-flags {
   fi
 
   echo "DOCKER_OPTS=\"${docker_opts} ${EXTRA_DOCKER_OPTS:-}\"" > /etc/default/docker
+
+  # If using a network plugin, extend the docker configuration to always remove
+  # the network checkpoint to avoid corrupt checkpoints.
+  # (https://github.com/docker/docker/issues/18283).
+  if [[ "${use_net_plugin}" == "true" ]]; then
```

Comment:
The corrupt docker network checkpoint can happen regardless whether you use the plugin. However, if you do use docker networking (i.e., don't use a separate network plugin), we can't remove the checkpoint arbitrarily without making sure that the checkpoint is corrupt. On the other hand, when you *do* use a network plugin (i.e., not relying on docker networking), the network checkpoints are useless to you and it's safe to be removed.